cves/CVE-2016-6186.yml

```diff
@@ -49,37 +49,42 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A pop-up related function was made vulnerable to cross site scripting
+  attacks due to the fact that the inputs that were being supplied by the user were
+  not being sanitized properly. This allowed attackers to enter dangerous scripts
+  which could steal authentication credentials from the user, and even change how
+  the site is being rendered to the user.
+
+  The fix to this issue was referncing the content in the text instead of the content
+  within the HTML, this prevents the execution of the data.
```

Comment:
Due to user supplied inputs not being sanitized properly.
Fix for this rather than to this.

Slight simplifications.

Reply:
Simplified it a bit